conf.py

```diff
@@ -854,7 +854,17 @@
 
 # A small copyright notice for the page footer (in HTML).
 # (translatable)
-CONTENT_FOOTER = 'Contents &copy; {date}         <a href="mailto:{email}">{author}</a> - Powered by         <a href="https://getnikola.com" rel="nofollow">Nikola</a>         {license}'
+CONTENT_FOOTER = '''
+Contents &copy; {date}         <a href="mailto:{email}">{author}</a> - Powered by         <a href="https://getnikola.com" rel="nofollow">Nikola</a>
+{license}
+<p>
+Сайт поддерживается российским сообществом Fedora, Red Hat не несет ответственности за его содержимое.<br>
+<a href="http://wiki.russianfedora.pro/index.php?title=%D0%91%D0%BB%D0%B0%D0%B3%D0%BE%D0%B4%D0%B0%D1%80%D0%BD%D0%BE%D1%81%D1%82%D0%B8">Благодарности</a> |
```

Comment:
shouldn't nikola automatically encode urls? I would prefer to have proper russian notation of URL ;)